lib/src/binary_codec.dart

```diff
@@ -305,5 +326,7 @@ class PostgresBinaryDecoder extends Converter<Uint8List, dynamic> {
     1184: PostgreSQLDataType.timestampWithTimezone,
     2950: PostgreSQLDataType.uuid,
     3802: PostgreSQLDataType.json,
+    46315: PostgreSQLDataType.geometry, /// TODO: Oid Changes on different databases after running `CREATE EXTENSION postgis`
+    46971: PostgreSQLDataType.geometry  /// `SELECT oid, typarray FROM pg_type WHERE typname in ('geometry','geography')`;
```

Comment:
@bettdouglas: If you have two databases on the same server, you create the postgis extension on both, will this be one entry or two for `'geometry'`?

Reply:
So, postgis has two general types which is 'geography' and 'geometry'. When you create the extension on one database, the two types('geography' and 'geometry') will be registered to the pg_type table. Running the create extension postgis again replicates the same behavior but doesn't affect types registered in other databases.

Reply:
In other words, it will have two new records per DB, and their OIDs will be different. It does complicate things, as we'll need a per-DB query (or another way to detect the type, which won't be cheap).

Reply:
That would be kind of expensive,but  i assume since the behaviour is different per database, and since every connection from dart has to specifies a specific database, the on connection query to get oids would just include one additional type 'geography' apart from 'geometry', or maybe I'm getting this wrong. or How were you planning to implement this?